index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Bengi's Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+</head>
+
+<body>
+    <main>
+        <header>
+            <nav>
+                <div id="logo">Bengi Genç</div>
+                <img id="burger" src="images/burger-menu-icon.png" alt="burger menu icon">
+            </nav>
+        </header>
+
+        <section id="personal-intro">
+            <h1>I’m Bengi <br>Glad to see you!</h1>
+            <p class="intro">I’m a software developer! I can help you build a product , feature or website Look through
+                some of my work and experience! If you like what you see and have a project you need coded, don’t
+                hestiate to contact me.</p>
+            <p class="connect">LET’S CONNECT</p>
```

Comment:
- [ ] Kindly change this into a link or a button as requested in the project requirements